### PR TITLE
OP-1002 [OH GUI] Add "createdBy" column in MovStockBrowser

### PR DESCRIPTION
**Pull request by mwithi**
See OP-1002.

### CODE REVIEW

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -447,7 +447,9 @@ image:image22.png[Pharmacy Menu,width=220,height=182]
 
 The *_Stock Movement Browser_* window shown below is opened.
 
-image:image23.png[Stock Movement Browser,width=598,height=327]
+image:image23.png[Stock Movement Browser]
+
+NOTE: The User column can be enabled or disabled by changing the _SINGLEUSER_ flag in the configuration file**.** Ask the Administrator to enable/disable this function or refer to the _Administrator’s Guide_.
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]: Is this correct?  `**.**`

**Review by dbmalkovsky — APPROVED** [review]
If my comment is off base then this is OK

File: doc_user/UserManual.adoc (hunk 1 above)
Comment (mwithi) [reply doc_user/UserManual.adoc]: nope... I'll fix it, thanks!

Comment (mwithi) [inline doc_user/UserManual.adoc]:
```suggestion
NOTE: The User column can be enabled or disabled by changing the _SINGLEUSER_ flag in the configuration file. Ask the Administrator to enable/disable this function or refer to the _Administrator’s Guide_.
```

**Pull request merged**